core/app/mailers/spree/order_mailer.rb

```diff
@@ -5,22 +5,22 @@ def confirm_email(order, resend = false)
       current_store = @order.store
       subject = (resend ? "[#{Spree.t(:resend).upcase}] " : '')
       subject += "#{current_store.name} #{Spree.t('order_mailer.confirm_email.subject')} ##{@order.number}"
-      mail(to: @order.email, from: from_address, subject: subject)
+      mail(to: @order.email, from: from_address, subject: subject, store_url: current_store.url)
```

Comment:
should we also reflect `store_url` to the view ?

Reply:
This is being used here: https://github.com/spree/spree/pull/10586/files#diff-c4f76565943d6afcaf07c2bdaa1c76b4da2945a6100f0b0be727e70288d0b388R26. It sets up `default_url_option` so it shouldn't be necessary to change anything on the view.

Reply:
@aplegatt I though about wrap this store name: https://github.com/spree/spree/blob/master/core/app/views/spree/order_mailer/confirm_email.html.erb#L22 with `store_url`